src/js/view/context-menu.js

```diff
@@ -59,7 +59,7 @@ silex.view.ContextMenu = function(element, model, controller) {
 
 };
 
-
+silex.view.ContextMenu.CLASS_NAME='silex-context-menu';
```

Comment:
the "@" keywords need to be on different lines and at the start of the line
you can look at how it is in the other parts of the code